src/azul/lambda_layer.py

```diff
@@ -23,8 +32,47 @@
 log = logging.getLogger(__name__)
 
 
+class Req(NamedTuple):
+    name: str
+    version: str
+    git: bool
+
+    wheel_re = re.compile(r'(?P<distribution>[^-]*)-'
+                          r'(?P<version>[^-]*)-'
+                          r'(?:(?P<build_tag>[^-]*)-)?'
+                          r'(?P<python_tag>[^-]*)-'
+                          r'(?P<abi_tag>[^-]*)-'
+                          r'(?P<platform_tag>[^-]*).whl')
```

Comment:
```suggestion
    wheel_re = re.compile(r'(?P<distribution>[^-]+)-'
                          r'(?P<version>[^-]+)-'
                          r'(?:(?P<build_tag>\d[^-]+)-)?'
                          r'(?P<python_tag>[^-]+)-'
                          r'(?P<abi_tag>[^-]+)-'
                          r'(?P<platform_tag>[^-]+).whl')
```
According to https://www.python.org/dev/peps/pep-0427/#file-name-convention the build tag must start with a number.

Reply:
Good catch.